internal/web_package/assembler.js

```diff
@@ -64,7 +64,7 @@ function main(params) {
         copy(path.join(f, file));
       }
     } else {
-      const content = fs.readFileSync(f, {encoding: 'utf-8'});
+      const content = fs.readFileSync(f);
       write(path.join(outdir, relative(f)), content);
```

Comment:
Cleaner to just use fs.copyFileSync(src, dest); (same a package.js)

Reply:
Oh.. but you need to also make sure the folder is there with `mkdirp(path.dirname(dest));`

Reply:
:+1: